tests/common.py

```diff
@@ -0,0 +1,42 @@
+
+"""Test generate script.
+"""
+
+
+import os
+from six import with_metaclass
+
+
+class HelpNotEmptyMeta(type):
+    """Test generator whether some help is empty."""
+
+    def __new__(meta, name, bases, dct):
+        """Create new test."""
+        def gen_test(a):
+            def test(self):
+                for line in self.HELPTEXT:
+                    if a in line:
+                        helpAtr = line.replace(a, '').strip()
+                        self.assertNotEqual(len(helpAtr), 0, 'Help in %s argument is empty.' % a)
+                        break
```

Comment:
A `self.assertTrue(False, '%s not in help text.' % a)` would be nice after the for-loop, to catch errors when an expected argument is missing.